source/apiVolontaria/volunteer/tests/tests_admin_Participation.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from django.contrib.auth.models import User
+from rest_framework.test import APITransactionTestCase
+
+from django.contrib.admin.sites import AdminSite
+from django.test import Client
+
+from ..factories import ParticipationFactory
+from ..admin import ParticipationAdmin
+from ..models import Participation
+
+
+class ParticipationTests(unittest.TestCase):
```

Comment:
These new tests have no direct link with the issue referenced in this PR. Please do not change other part of the system without special issue.